src/VisualStudio/Core/Def/Implementation/Library/FindResults/TreeItems/ExternalTreeItem.cs

```diff
@@ -9,14 +9,14 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Library.FindResults
 {
-    internal class ExternalListItem : AbstractListItem
+    internal class ExternalTreeItem : AbstractTreeItem
```

Comment:
From the use above, it means "From outside Roslyn", as in, when the Xaml language service gives us definitions in Xaml files, etc.